test/attributes.test

```diff
@@ -52,9 +52,15 @@ hi\{key="abc{#hi}"
 ```
 
 ```
-hi{key="\{#hi"}
+hi{key="\"#hi"}
 .
-<p><span key="{#hi">hi</span></p>
+<p><span key="&quot;#hi">hi</span></p>
+```
+
+```
+hi{key="hi\"#hi"}
+.
+<p><span key="hi&quot;#hi">hi</span></p>
```

Comment:
No, this last one is testing a slightly different case from the "\"#hi", one.  The "hi\"#hi" one tests that unescaped text can occur before the escape, such that the sections before and after the escape are preserved.  I was looking at someones parser, and I didn't see how their implementation could handle this case, since the parsing machinery appeared to only handle contiguous strings.